crates/net/eth-wire-types/src/message.rs

```diff
@@ -229,11 +234,13 @@ pub enum EthMessage<N: NetworkPrimitives = EthNetworkPrimitives> {
         serde(bound = "N::Receipt: serde::Serialize + serde::de::DeserializeOwned")
     )]
     Receipts(RequestPair<Receipts<N::Receipt>>),
+    /// Represents a `Unkown` request-response
```

Comment:
we should document here that this is still the encoded message and not mention request-response